doc/rbd/rbd-config-ref.rst

```diff
@@ -134,3 +134,54 @@ Read-ahead is automatically disabled if caching is disabled.
 :Type: 64-bit Integer
 :Required: No
 :Default: ``50 MiB``
+
+
+RBD Default Features
+========================
```

Comment:
Nit: the underlining should match in length to its text

Reply:
Sure i'll modify it.